src/cli/miner_cli_info.erl

```diff
@@ -14,6 +14,7 @@
 
 -define(DEFAULT_LOG_HIT_COUNT, "5").                                                        %% the default number of error log hits to return as part of info summary
 -define(DEFAULT_LOG_SCAN_RANGE, "500").                                                     %% the default number of error log entries to scan as part of info summary
+-define(ONBOARDING_API_URL_BASE, "https://onboarding.dewi.org/api/v2").
```

Comment:
I'm not sure if you checked in with others on this, but I would imagine the idea of having the Miner depend on ETL for anything is probably not desirable.

Reply:
I agree am happy to report that this change doesn't make anything in the miner depend on this server. It's a convenience function in the miner CLI to allow a user or script to determine the maker "address" so that they can construct a proper `assert_gateway` or `assert_location` transaction correctly when using the `txn` CLI verb.

I first proposed having this feature be a part of the wallet CLI, but @madninja hinted that it should be in the miner CLI. So here it is.

Reply:
And just to be super clear, this endpoint is only queried when a user asks the miner to do so from the command line. It is not queried during the miner boot process nor consulted at all during regular mining tasks.